packages/lib/src/components/core/lume-chart/lume-chart.vue

```diff
@@ -438,6 +439,7 @@ function handleInternalHover(index: number) {
         : allOptions.value.tooltipOptions.targetElement;
     showTooltip(targetElement);
   }
+  emit('group-mouseenter', index);
```

Comment:
These events should have a name related to what they're attached to - something like `emit('hovered-index-changed', newIndex, oldIndex)` - naming them after the groups is misleading 🤔

Reply:
yeah, makes sense to have it attached based on what they're attached to!
will update